### PR TITLE
csv link in the ui

### PR DESCRIPTION
**Pull request by whaught**
## Proposed Changes

* https://github.com/google/exposure-notifications-verification-server/pull/1691 created the routes to do CSV marshal and this sticks the link into the UI
    * (sorry for the absurdly small PRs - I missed a spot here)
* `target="_blank"` for new tab when downloading as a file

### CODE REVIEW

File: cmd/server/assets/apikeys/show.html [hunk 1]
```diff
@@ -86,8 +86,8 @@ <h1>{{$authApp.Name}} API key</h1>
         </span>
         <span>
           <span class="mr-1">Export as:</span>
-          <a href="/stats/realm/api-keys/{{$authApp.ID}}.csv" class="mr-1">CSV</a>
-          <a href="/stats/realm/api-keys/{{$authApp.ID}}.json">JSON</a>
+          <a href="/stats/realm/api-keys/{{$authApp.ID}}.csv" target="_blank" class="mr-1">CSV</a>
+          <a href="/stats/realm/api-keys/{{$authApp.ID}}.json" target="_blank">JSON</a>
```
Comment (sethvargo) [inline cmd/server/assets/apikeys/show.html:90]: Why target blank? The CSV should trigger as a file download.

Comment (whaught) [reply cmd/server/assets/apikeys/show.html:90]: That's true it may be redundant for CSV, but for JSON it takes over the window

**Review by sethvargo — APPROVED** [review]
/lgtm

**Pull request merged**